src/figuresClasses.ts

```diff
@@ -1,19 +1,101 @@
 export interface Figure {
-
+  shape: string
+  color: string
+  getArea(): number | never
+  sides?: number[]
+  radius?: number
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: string;
+
+  color: string;
+
+  sides: number[];
+
+  getArea: () => number | never;
+
+  constructor(color: string, ...sides: number[]) {
+    this.shape = 'triangle';
+    this.color = color;
+    this.sides = sides;
+
+    const [a, b, c] = sides;
+    let p = a + b + c;
 
+    const errorValue = sides.some((elem: number) => elem <= 0
+        || p - elem <= elem);
+
+    if (errorValue) {
+      throw new Error("sides 1, 2 and 3 can't form a triangle");
+    }
+
+    this.getArea = (): number | never => {
```

Comment:
getArea method should always return the number of area, why did you use never type here?